phnt/include/ntldr.h

```diff
@@ -1384,57 +1448,20 @@ LdrSetImplicitPathOptions(
     );
 #endif
 
-#if (PHNT_VERSION >= PHNT_WINDOWS_10)
-#ifdef PHNT_INLINE_TYPEDEFS
-/**
- * The LdrControlFlowGuardEnforced function checks if Control Flow Guard is enforced.
- *
- * @return BOOLEAN TRUE if Control Flow Guard is enforced, FALSE otherwise.
- */
-FORCEINLINE
-BOOLEAN
-NTAPI
-LdrControlFlowGuardEnforced(
-    VOID
-    )
-{
-    return LdrSystemDllInitBlock.CfgBitMap && (LdrSystemDllInitBlock.Flags & 1) == 0;
-}
-#else
-// rev
+#if (PHNT_VERSION >= PHNT_WINDOWS_10RS3)
+// private
 /**
  * The LdrControlFlowGuardEnforced function checks if Control Flow Guard is enforced.
  *
  * @return BOOLEAN TRUE if Control Flow Guard is enforced, FALSE otherwise.
```

Comment:
The documentation needs updated. 

Looks like `al` isn't in routine or by the call sites in ntdll, they all operate with `eax`. So I agree that this probably shouldn't have been `BOOLEAN`. OTOH `BOOLEAN` seems convenient since only the LSB is ever relevant according to the disassembly :shrug:.